builtin/common/misc_helpers.lua

```diff
@@ -486,6 +486,17 @@ function table.copy(t, seen)
 end
 
 
+function table.len(t)
+    local counter = 0
```

Comment:
I don't understand why when I upload my code on github, all tabs are displayed incorrectly. In my original code the tabs look like aligned with other parts of the file.

Reply:
https://dev.minetest.net/Lua_code_style_guidelines
Indent with tabs.

Reply:
I suppose two tabs?